### DISCUSSION TITLE
template repositories

### DISCUSSION THREAD

**Discussion by Aariq** [SORTEE-Github-Hackathon/manuscript#108] [Ideas]
Something I don't think we've talked about in the manuscript, but I think would be useful for EEB researchers is the ability to create [template repositories](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-template-repository) with GitHub. Templates (e.g. https://github.com/Aariq/paper-template) replace the "clone" button with a "use this template" button.  One possible use would be for a lab organization to have templates for research compendia or manuscripts with all the backbone infrastructure already set up.  That way, new trainees don't need to master everything about reproducibility or lab standards or whatever---it's already set up for them.

**Comment by vjf2** [comment]
+1 Definitely, we do that in my lab and it works great. I think this is mentioned at the end of the "GitHub Organizations" section but maybe could be expanded upon? @katherinehebert @colebrookson

**Reply by Aariq** [reply]
Yeah, I think that's a natural place to discuss it, but templates can also be used by individuals just to save time setting up projects for example.
